glossary/ja/terms/creative-commons/index.md

```diff
@@ -0,0 +1,9 @@
+---
+section: terms
+lang: ja
+title: Creative Commons
+---
+
+クリエイティブ・コモンズ。幾多の標準的な{ライセンス}によりパブリッシュされたコンテンツの再利用を促進するために、2001年に設立された非営利組織であり、オープンなものもあり（他に{非商用}条項を含むものもあるが）、コンテンツを[再利用して](/glossary/ja/terms/re-use/)リリースする際に利用することができ、合わせてその意味するところを明確にできる。
+
+A non-profit organisation founded in 2001 that promotes re-usable content by publishing a number of standard {licences}, some of them open (though others include a {non-commercial} clause), that can be used to release content for [re-use](/glossary/en/terms/re-use/), together with clear explanations of their meaning.
```

Comment:
Hi guys, notice that the link here [re-use] link to the English version, do you want to link it to the Japanese one?